modules/distribution/src/assembly/bin.xml

```diff
@@ -1111,6 +1111,13 @@
             <outputDirectory>${pom.artifactId}-${pom.version}/lib/</outputDirectory>
             <includes>
                 <include>org.slf4j:slf4j-simple</include>
+                <include>io.netty:netty-handler</include>
```

Comment:
why we need need this here

Reply:
If these libs required for osgi services we need to add this as orbit bundles.

Reply:
Removed from here and added to orbit